python/note_input_widget/ui/note_input_widget.py

```diff
@@ -145,77 +146,61 @@ def setupUi(self, NoteInputWidget):
         self.attachment_list_tree.headerItem().setText(0, "1")
         self.attachments_list_layout.addWidget(self.attachment_list_tree)
         self.horizontalLayout_4 = QtGui.QHBoxLayout()
-        self.horizontalLayout_4.setSpacing(3)
+        self.horizontalLayout_4.setSpacing(7)
+        self.horizontalLayout_4.setContentsMargins(-1, 3, -1, 3)
         self.horizontalLayout_4.setObjectName("horizontalLayout_4")
         self.add_button = QtGui.QToolButton(self.attachment_list)
-        icon4 = QtGui.QIcon()
-        icon4.addPixmap(QtGui.QPixmap(":/tk_framework_qtwidgets.note_input_widget/plus.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
-        self.add_button.setIcon(icon4)
+        icon2 = QtGui.QIcon()
+        icon2.addPixmap(QtGui.QPixmap(":/tk_framework_qtwidgets.note_input_widget/plus.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
+        self.add_button.setIcon(icon2)
         self.add_button.setAutoRaise(True)
         self.add_button.setObjectName("add_button")
         self.horizontalLayout_4.addWidget(self.add_button)
         self.remove_button = QtGui.QToolButton(self.attachment_list)
-        icon5 = QtGui.QIcon()
-        icon5.addPixmap(QtGui.QPixmap(":/tk_framework_qtwidgets.note_input_widget/minus.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
-        self.remove_button.setIcon(icon5)
+        icon3 = QtGui.QIcon()
+        icon3.addPixmap(QtGui.QPixmap(":/tk_framework_qtwidgets.note_input_widget/minus.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
```

Comment:
What are the plus/minus buttons for? When reviewing the before/after screenshots, I was confused with these.

Reply:
The plus/minus buttons only show when adding attachments to the note. You can use the "plus" icon to add an attachment, then you can select the attachment after it has been added and click the "minus" icon to remove it.

Reply:
Ok so why do we see the minus button with the plus one when there is no attachment (one of your screenshot above shows that). 
I know you did not introduce that here but would that be something to clean up here? Or am I missing something else?

Reply:
That is just how the UI is currently. We can probably disable the button, when there are no attachments to remove

Reply:
@julien-lang see description with updated screenshots for "-" button disabled when no attachment is selected to remove